test/nvexec/variant.cpp

```diff
@@ -70,26 +70,27 @@ __global__ void kernel(V* v, T alt) {
 
 TEST_CASE("variant emplaces alternative from GPU", "[cuda][stream][containers][variant]") {
   using variant_t = variant_t<int, double>;
-  variant_t v{};
-  THROW_ON_CUDA_ERROR(cudaHostRegister(&v, sizeof(variant_t), cudaHostRegisterDefault));
+  variant_t *v{};
+  THROW_ON_CUDA_ERROR(cudaMallocManaged(&v, sizeof(variant_t)));
+  new (v) variant_t();
```

Comment:
Does clangd not report the `clang(need_header_before_placement_new)` error here for you @senior-zero?

Reply:
It does, but inclusion of `new` doesn't fix the issue for me.